packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx

```diff
@@ -90,6 +148,8 @@ export function PreviewCard(props: PreviewCardProps) {
         retryMaskBoxPurchasedTokens,
     } = useContainer(Context)
 
+    const [currentTab, onChange, tabs] = useTabs('Articles', 'Details')
```

Comment:
i18n

Reply:
unnecessary

Reply:
the input for this hook is not for display, but more like an enum. therefore it does not need i18n